src/App.tsx

```diff
@@ -26,70 +26,128 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  if (sortType === SortType.ALPHABET) {
+    visibleGoods.sort((a, b) => {
+      if (a > b) {
+        return 1;
+      }
+
+      if (a < b) {
+        return -1;
+      }
+
+      return 0;
+    });
+  }
+
+  if (sortType === SortType.LENGTH) {
+    visibleGoods.sort((a, b) => {
+      return a.length - b.length;
+    });
+  }
+
+  if (isReversed) {
+    visibleGoods.reverse();
+  }
 
   return visibleGoods;
 }
 
-// DON'T save goods to the state
-// type State = {
-//   isReversed: boolean,
-//   sortType: SortType,
-// };
-
-export const App: React.FC = () => {
-  return (
-    <div className="section content">
-      <div className="buttons">
-        <button
-          type="button"
-          className="button is-info is-light"
-        >
-          Sort alphabetically
-        </button>
-
-        <button
-          type="button"
-          className="button is-success is-light"
-        >
-          Sort by length
-        </button>
-
-        <button
-          type="button"
-          className="button is-warning is-light"
-        >
-          Reverse
-        </button>
-
-        <button
-          type="button"
-          className="button is-danger is-light"
-        >
-          Reset
-        </button>
-      </div>
+type State = {
+  isReversed: boolean,
+  sortType: SortType,
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    isReversed: false,
+    sortType: SortType.NONE,
+  };
+
+  componentDidMount = () => {
+    const buttons = document.querySelector('.buttons');
+
+    buttons?.addEventListener('click', ({ target }) => {
+      if (target) {
+        const button = target as HTMLButtonElement;
+
+        if (buttons) {
+          Array.from(buttons.children).forEach(el => {
+            el.classList.add('is-light');
+          });
+        }
+
+        if (button.matches('.button')) {
+          button.classList.remove('is-light');
+        }
+      }
+    });
```

Comment:
Remove this. We did not work with DOM in react. 
Add required class names in render() method